keps/sig-cli/3659-kubectl-apply-prune/README.md

```diff
@@ -479,26 +394,26 @@ treat this manifest as invalid and return an error.
 Although ApplySet parent objects can (in theory) be of any type, for both performance and simplicity, we choose to limit supported types to the following:
 - ConfigMap
 - Secret
-- custom resources, where the CRD registering them is labeled with `applyset.k8s.io/role/parent` (The value is currently ignored, but implementors should set an empty value to be forwards-compatible with future evolution of this convention.)
+- custom resources, where the CRD registering them is labeled with `applyset.kubernetes.io/is-parent-type` (The value is currently ignored, but implementors should set it to "true" to be forwards-compatible with future evolution of this convention.)
```

Comment:
The docs PR currently instructs people to set it to "true". I think that is the logical default value given the name we ended up going with.